### PR TITLE
fix: iOS Nullable

### PR DESCRIPTION
**Pull request by workgroupengineering**
## What does the pull request do?

fixes:
```bash
Warning	CS8601	Possible null reference assignment.	Avalonia.iOS	src\iOS\Avalonia.iOS\Storage\IOSStorageItem.cs	27	N/A
Warning	CS8602	Dereference of a possibly null reference.	Avalonia.iOS	src\iOS\Avalonia.iOS\Storage\IOSStorageItem.cs	27	N/A
Warning	CS8601	Possible null reference assignment.	Avalonia.iOS	src\iOS\Avalonia.iOS\Storage\IOSStorageItem.cs	28	N/A
Warning	CS8602	Dereference of a possibly null reference.	Avalonia.iOS	src\iOS\Avalonia.iOS\Storage\IOSStorageItem.cs	28	N/A
Warning	CS8603	Possible null reference return.	Avalonia.iOS	src\iOS\Avalonia.iOS\TextInputResponder.cs	96	N/A
Warning	CS8603	Possible null reference return.	Avalonia.iOS	src\iOS\Avalonia.iOS\TextInputResponder.cs	384	N/A
```

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by workgroupengineering** [comment]
It's a draft because I don't have an iOS device to test this PR.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0029513-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

File: src/iOS/Avalonia.iOS/PlatformSettings.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public override PlatformColorValues GetColorValues()
             ColorContrastPreference.High :
             ColorContrastPreference.NoPreference;
 
-        UIColor? tintColor = null;
+        UIColor tintColor = null;
```
Comment (maxkatz6) [inline src/iOS/Avalonia.iOS/PlatformSettings.cs:26]: I think nullable can be easily enabled in this file.
Either way, let's not remove "?" from code, better to silence this specific warning or even better complete nullability in the file.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0030114-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0030205-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Pull request merged**